FEATURES.md

```diff
@@ -56,6 +56,8 @@ VVVVVV: Community Edition has accepted contributions from Misa, AllyTally, leo60
 
 - The game no longer crashes if you give an invalid filename to a command
 
+- Added being able to use `sprites2.png`, `sprites3.png`, `sprites4.png`, etc. and using Shift+F9 to switch between them in the editor, not yet showing up in the editor
+
```

Comment:
This should be added to the in-game changelog as well, which is in `titlerender.cpp`.